src/objects/button-group/button-group.stories.mdx

```diff
@@ -0,0 +1,71 @@
+import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
+import demo from './demo/demo.twig';
+
+<Meta
+  title="Objects/Button Group"
+  argTypes={{
+    tight: {
+      type: { name: 'boolean' },
+      defaultValue: false,
+      description: 'Controls the tight-spacing modifier',
+    },
```

Comment:
Is this still relevant?

Reply:
Oops, good catch! 🔍 👀

Reply:
Removed. https://github.com/cloudfour/cloudfour.com-patterns/pull/1240/commits/eec8bd3ed1c465a5be4c8c08cc5a297e9078611e